CodeDrivers/CodeDrivers/CodeDrivers/CsvHanlder.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeDrivers
+{
+    internal class CsvHanlder
+    {
+        private string filePath = @"C:\Users\Janek\Desktop\FakeCrudentials.csv";
```

Comment:
To zadziała tylko u Ciebie na kompie (wątpię, że każdy użytkownik apki będzie miał folder Janek na kompie :) ). Lepiej podać ścieżkę względną, np: "csv\FakeCrudentials.csv" wtedy program będzie oczekiwał, że plik znajduje się w bin/Debug/csv i to zadziała na dowolnym kompie